torch_geometric/nn/conv/cheb_conv.py

```diff
@@ -113,6 +111,13 @@ def __norm__(self, edge_index, num_nodes: Optional[int],
                                                 normalization, dtype,
                                                 num_nodes)
 
+        if lambda_max is None:
+            lambda_max = 2.0 * edge_weight
+        if not isinstance(lambda_max, torch.Tensor):
```

Comment:
```suggestion
        elif not isinstance(lambda_max, torch.Tensor):
```

Reply:
I preserved this if statement because the original code included it.
But since `lambda_max` is an OptTensor, it is either `None` or an instance of `torch.Tensor`.
So I think it could be deleted.